libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -906,20 +913,34 @@ - (void)oauthCoordinatorDidFetchAuthCode:(SFOAuthCoordinator *)coordinator authI
 }
 
 - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator didBeginAuthenticationWithView:(WKWebView *)view {
-
     SFLoginViewController *loginViewController = [self createLoginViewControllerInstance:coordinator];
     loginViewController.oauthView = view;
     SFSDKAuthViewHolder *viewHolder = [SFSDKAuthViewHolder new];
     viewHolder.loginController = loginViewController;
     viewHolder.scene = coordinator.authSession.oauthRequest.scene;
+    
+    void (^authViewDisplayBlock)(void) = ^{
+        
+        self.authViewHandler.authViewDisplayBlock(viewHolder);
+        if (!coordinator.overridingFrontDoorBridgeUrlMatchesConsumerKey) {
+            UIAlertController* alertController = [UIAlertController
+                                                  alertControllerWithTitle:@"Error"
+                                                  message:@"Cannot use another app's login QR Code.  Please log in to this app."
```

Comment:
We shouldn't hardcode the message. Instead it should come from [Localizable.strings](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/shared/resources/SalesforceSDKResources.bundle/en.lproj/Localizable.strings)